sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -104,6 +104,17 @@ public DynamoDBContextConfig()
         /// .NET and DynamoDB types happens.
         /// </summary>
         public DynamoDBEntryConversion Conversion { get; set; }
+
+        /// <summary>
+        /// Skip the internal DescribeTable call that caches table metadata. Table metadata must be 
+        /// defined by <see cref="DataModel.DynamoDBAttribute"/> attributes and/or in <see cref="AWSConfigsDynamoDB"/>.
+        /// </summary>
+        /// <remarks>
+        /// Setting this to true can avoid latency and potential deadlocks due to the internal DescribeTable call that is used to populate the SDK's cache of 
+        /// table medata. It requires that the table's index schema be fully described via the above methods, otherwise exceptions may be thrown and/or 
+        /// the results of certain DynamoDB operations may change. It is recommended to test your application prior to setting it to true in production code.
+        /// </remarks>
+        public bool? SkipDescribeTableLoad { get; set; }
```

Comment:
Curious what other reviewers think, but I'm still not too attached to `SkipDescribeTableLoad`. It might be too from-our-perspective/internal, since some users may not realize that that call is happening.

That said, not I don't have a better alternative.

Reply:
Deferring to @normj for the naming scheme

Reply:
Renamed the property to `DisableFetchingTableMetadata`